packages/architectura/README.md

```diff
@@ -0,0 +1,141 @@
+# Architectura
+
+Architectura is a light-weight framework using no external dependency.
+It facilitates the implementation of a Domain Driven Design software architecture.
+It provides a simple development API and blazing fast performances.
+
+## Forewords - Read first
+
+Architectura is an opinionated, non-intrusive framework.
+
+### Opinionated
+
+Architectura is opinionated because it follows and incites users to follow a Domain Drive Design software architecture.
+It is also opinionated because it is dependency-free (outside of the Vitruvius Labs ecosystem) and enforces extremely strict typing.
+The opinion of the team behind Architectura is top provide a framework with premium quality standards.
+Architectura also follows the SOLID principles; however, we do not believe that dependency injection is the appropriate answer to the Dependency-inversion principle. To that end, Architectura does not embark any dependency-injection tool.
+
+> [!IMPORTANT]
+> Note regarding dependency-injection.
+> Whilst you can add any dependency-injection third party package, we invite you to reconsider before doing so.
+> Dependency-injection has many downsides and pitfalls that are often brushed over. This can rapidly make the technical debt of your project spiral out of control.
+> We encourage you to learn how to implement dependency-inversion in a more conventional, rigorous way than rely on magic.
+> Again, Architectura does not enforce this idea on you, you are still free to use any package that you consider appropriate to your project.
+
+### Non-intrusive
+
+Architectura is non-intrusive because you can easily onboard yourself on it and offboard.
+
+> Customers stick with terrible, non-functional solutions because they are prisoners of it. It is a form of Stockholm Syndrome.
+> \- *Nicolas "SmashingQuasar" Lebacq, co-creator of Architectura*
+
+The objective is not to retain the user base by introducing unique patterns, a large number of dependencies or even an interlinking of services.
+The objective is to convince the user base that Architectura is the right fit for their project and needs. If they realise it wasn't due to their software needs sliding out of the scope of Architectura, they are not prisoners and can easily offboard themselves from it.
+
+## Domains structure
+
+Architectura articulates around the use of domains.
+As with most feature within Architectura, you are not forced to use domains. You are instead encouraged to do so to help structure your code base.
+
+> [!IMPORTANT]
+> Explaining what a domain is is out of the scope of this documentation. If you would like to know more about domains in the context of Domain Driven Design, please refer to the reference book from Eric Evans called *Domain-Driven Design: Tackling Complexity in the Heart of Software*.
```

Comment:
`Domain-Driven` instead of `Domain Driven`. In general, I've only seen it hyphenated.